common/azure-common/src/main/java/com/azure/common/implementation/RestProxy.java

```diff
@@ -252,22 +247,18 @@ private HttpRequest configRequest(HttpRequest request, SwaggerMethodParser metho
             if (isJson) {
                 final String bodyContentString = serializer.serialize(bodyContentObject, SerializerEncoding.JSON);
                 request.withBody(bodyContentString);
-            }
-            else if (FluxUtil.isFluxByteBuf(methodParser.bodyJavaType())) {
+            } else if (FluxUtil.isFluxByteBuf(methodParser.bodyJavaType())) {
                 // Content-Length or Transfer-Encoding: chunked must be provided by a user-specified header when a Flowable<byte[]> is given for the body.
                 //noinspection ConstantConditions
                 request.withBody((Flux<ByteBuf>) bodyContentObject);
-            }
-            else if (bodyContentObject instanceof byte[]) {
+            } else if (bodyContentObject instanceof byte[]) {
                 request.withBody((byte[]) bodyContentObject);
-            }
-            else if (bodyContentObject instanceof String) {
+            } else if (bodyContentObject instanceof String) {
                 final String bodyContentString = (String) bodyContentObject;
                 if (!bodyContentString.isEmpty()) {
                     request.withBody(bodyContentString);
                 }
-            }
-            else {
+            } else {
```

Comment:
You should avoid the desire to make unrelated changes when submitting a PR. These kind of formatting changes should ideally not be included in a PR for a new feature.

Reply:
Reverted these types of changes out, I left the import sorting and removing changes though